### DISCUSSION TITLE
Is src directory possible?

### DISCUSSION THREAD

**Discussion by o-az** [freshframework/fresh#253] [Q&A]
Moved `islands`, `routes`, `fresh.gen.ts`, `dev.ts` and `main.ts` to `src` directory.

Updated `start` task in `deno.json` to: `deno run -A --watch=src/static/,src/routes/,src/islands/ --no-check src/dev.ts`.

Project runs and routes work but this error is consistently logged:
![CleanShot 2022-06-18 at 08 40 01](https://user-images.githubusercontent.com/23618431/174440261-6b6b0b03-9c12-4ecb-a3bb-2a025701e871.png)

I'm trying to avoid this error. Is there a proper way to have a `.src/` directory or is it not possible as of now?

**Comment by lucacasonato** [comment]
This should work. Moved it as an issue to https://github.com/lucacasonato/fresh/issues/257

**Reply by PhoneDroid** [reply]
Please close, issue has been resolved, 
solution ( docs addition ) removes the 
need for this to stay open.
